realsense2_description/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.5)
+project(realsense2_description)
+
+find_package(ament_cmake REQUIRED)
+
+find_package(realsense2_camera REQUIRED)
+find_package(realsense2_node REQUIRED)
```

Comment:
are these packages actually used in the realsense2_description package? (similar comment in package.xml)